packages/integrations/src/vue.ts

```diff
@@ -1,103 +1,298 @@
 import { EventProcessor, Hub, Integration } from '@sentry/types';
-import { getGlobalObject, isPlainObject, logger } from '@sentry/utils';
+import { basename, getGlobalObject, logger, timestampWithMs } from '@sentry/utils';
+import { Integrations as APMIntegrations, Span as SpanClass } from '@sentry/apm';
+
+interface IntegrationOptions {
+  Vue: any;
+  /**
+   * When set to false, Sentry will suppress reporting of all props data
+   * from your Vue components for privacy concerns.
+   */
+  attachProps: boolean;
+  /**
+   * When set to true, original Vue's `logError` will be called as well.
+   * https://github.com/vuejs/vue/blob/c2b1cfe9ccd08835f2d99f6ce60f67b4de55187f/src/core/util/error.js#L38-L48
+   */
+  logErrors: boolean;
+  tracing: boolean;
```

Comment:
Can we reasonably detect that tracing is enabled without this?
We've talked about enabling tracing whenever the Tracing integration was on. That would make for one less boilerplate line in `Sentry.init`.

Reply:
Sure, I'm already doing that. But what if someone wants to have error handling, xhr tracing, but skip Vue tracing?

Reply:
We also cannot tell if `Tracing` integration is in place during setup (subject to change in v6)